### DISCUSSION TITLE
How to make device shock on GTS mini 4 ZeppOS1.0 ?

### DISCUSSION THREAD

**Discussion by blockmedia** [zepp-health/zeppos-forum#167] [Q&A]
The ZeppOS1.0 device app does not seem to directly support the ability to make the watch produce a shock.
Have any advice~

deeeeeeply appreciate

**Comment by blockmedia** [comment]
sorry, miss VIBRATE sensor

**Reply by liqin-ZeppHealth** [reply]
yes, you can you vibrate sensor for your need.
